hri_safety_sense/config.yaml

```diff
@@ -0,0 +1,4 @@
+VscProcess:
```

Comment:
I missed this before, but a more descriptive name for the node (here and in the constructor) would be helpful.  Just imagine looking at the output of `ros2 node list`; it wouldn't be clear that this is the HRI driver.  Something like `hri_joystick_node` or something along those lines seems more descriptive to me.

Reply:
Changed node name and compile target to `hri_joystick_node` in d580097